UniConnectApp/app/src/test/java/com/example/uniconnect/UniConnectUnitTest.kt

```diff
@@ -1,105 +1,19 @@
 package com.example.uniconnect
 
-import androidx.arch.core.executor.testing.InstantTaskExecutorRule
-import androidx.lifecycle.Observer
-import com.example.uniconnect.dto.Post
 import com.example.uniconnect.dto.University
-import com.example.uniconnect.service.IUniversityService
-import com.example.uniconnect.service.UniversityService
-import io.mockk.MockKAnnotations
-import io.mockk.coEvery
-import io.mockk.impl.annotations.MockK
+import org.junit.Assert.assertTrue
 import org.junit.Test
-import kotlinx.coroutines.Dispatchers
-import kotlinx.coroutines.launch
-import kotlinx.coroutines.newSingleThreadContext
-import kotlinx.coroutines.test.resetMain
-import kotlinx.coroutines.test.runTest
-import kotlinx.coroutines.test.setMain
-import org.junit.After
-
-import org.junit.Assert.*
-import org.junit.Before
-import org.junit.Rule
-import org.junit.rules.TestRule
-import java.util.concurrent.CountDownLatch
-import java.util.concurrent.TimeUnit
 
 /**
  * Example local unit test, which will execute on the development machine (host).
  *
  * See [testing documentation](http://d.android.com/tools/testing).
  */
 class UniConnectUnitTest {
-
-    @get:Rule
-    var rule: TestRule = InstantTaskExecutorRule()
-
-    lateinit var mvm : MainViewModel
-
-    private val mainThreadSurrogate = newSingleThreadContext("Main thread")
-
-    @MockK
-    lateinit var mockUniService : UniversityService
-
-
-    lateinit var universityService: IUniversityService
-    var allUniversities : List<University>? = ArrayList<University>()
```

Comment:
none of these things were actually being used, leaving a lot of code sitting around, not contributing

Reply:
Are you certain they are not being used?
It looks like a valid unit test to me.